src/elements/pages/dev/game/pages/game-overview.ts

```diff
@@ -239,14 +235,17 @@ export default class DevGameOverview extends LitElement {
 				versionId: this.game.versions[0].versionId
 			});
 
-			this.namespaceModalClose();
+			// Wait until a new update on the game stream before navigating
```

Comment:
Why isn't the namespace accessible when `createGameNamespace` returns a 200? The "namespace not found" is a server-side error, not a client-side error, right?

Reply:
No, it has to do with the client not having the info to display the navbar and other navigation elements. The error actually comes from `src/elements/pages/dev/game/pages/rvt-game-dashboard.ts:63`